src/Azure/Azure.Quantum.Client/SubmitterFactory.cs

```diff
@@ -28,6 +28,11 @@ public static class SubmitterFactory
                 "Microsoft.Quantum.Providers.Targets.MicrosoftSimulatorSubmitter, Microsoft.Quantum.Providers.Core",
                 "QirSubmitter",
                 ImmutableArray.Create("FullComputation")),
+            new SubmitterInfo(
```

Comment:
I would suggest separating the patterns for `microsoft.sim.*` targets and `microsoft.estimator` since their configuration is different.

Reply:
Done! I changed to be specific to microsoft.estimator(.*)